src/main/java/org/gridsuite/network/map/NetworkMapService.java

```diff
@@ -92,7 +92,7 @@ private List<String> getVoltageLevelsIds(UUID networkUuid, String variantId, @No
     }
 
     public List<ElementInfos> getVoltageLevelEquipments(UUID networkUuid, String voltageLevelId, String variantId, @NonNull List<String> substationsId) {
```

Comment:
substationsId is never sent. You can remove it and clean the front (https://github.com/gridsuite/gridstudy-app/blob/a18608568a624027ca069d096a1ab645ab3183ff/src/services/study/network-map.ts#L70) and study-server